README.md

```diff
@@ -1,16 +1,16 @@
-# IOVNS
-This repo contains bnsd built on top of the cosmos-sdk
+# IOV Name Service (IOVNS)
+This repo contains the IOV Name Service daemon (iovnsd) and command-line interface (iovnscli) built on top of the cosmos-sdk.
 
-Everything is still very very very experimental, and I expect the design to dramatically change over the course of the coming days.
-
-## Running docker image
+## Running via docker image
 ```shell script
+# make the apps
+make build
 # build docker script
 docker build -t iovns .
 # initialize chain
 bash scripts/init.sh
 # run docker iovnsd
-docker run -it -p 127.0.0.1:26657:26657 -p 127.0.0.1:26656:26656 \
-  -v ~/.iovnsd:/app/.iovnsd -v ~/.iovnscli:/app/.iovnscli \ 
+docker run -it -p 127.0.0.1:46657:26657 -p 127.0.0.1:46656:26656 \
+  -v ~/.iovnsd:/app/.iovnsd -v ~/.iovnscli:/app/.iovnscli \
```

Comment:
@fdymylja , when I try to run docker I get the following error:
```
[dave@debt iovns]$ docker run -it -p 127.0.0.1:46657:26657 -p 127.0.0.1:46656:26656 \
>   -v ~/.iovnsd:/app/.iovnsd -v ~/.iovnscli:/app/.iovnscli \
>   iovns iovnsd start
ERROR: Config File "app" Not Found in "[/app/.iovnsd /app/.iovnsd/config]"
```
The app config directories exists and look good to me:
```
dave@debt iovns]$ ls -lh ~/.iovnsd ~/.iovnscli
/home/dave/.iovnscli:
total 8.0K
drwxrwxr-x. 2 dave dave 4.0K Jun 12 12:03 config
drwx------. 2 dave dave 4.0K Jun 12 12:03 keyring-test-iovns

/home/dave/.iovnsd:
total 8.0K
drwx------. 3 dave dave 4.0K Jun 12 12:03 config
drwx------. 2 dave dave 4.0K Jun 12 12:03 data
```
What am I doing wrong?

Reply:
Uhm, it's an interesting error, it appears to me that iovnsd is trying to find an "app" config file, AFAIK there should not be any config called "app". Maybe @orkunkl can help you better than I can since he took care of setting up iovns for k8s so I think he's more aware of configuration settings and how they should be passed as parameters to the app.

Reply:
Solved with the latest commit

Reply:
Great catch!